openreview/conference/builder.py

```diff
@@ -340,6 +343,10 @@ def set_submission_revision_stage(self, stage):
 
     def set_decision_stage(self, stage):
         self.decision_stage = stage
+
+        if self.decision_stage.decisions_file is not None:
```

Comment:
Looks like we are trying to post decisions before running `__create_decision_stage()`, this will give an error if the decision invitations haven't been created yet (or if they run the decision stage again to make a change to the decision options for example)